lang/ru/lang.php

```diff
@@ -16,17 +16,27 @@
         'access_posts' => 'Управление записями блога',
         'access_categories' => 'Управление категориями блога',
         'access_other_posts' => 'Управление записями других пользователей',
+        'access_import_export' => 'Разрешено импортировать и экспортировать записи',
+        'access_publish' => 'Разрешено публиковать записи',
+        'manage_settings' => 'Управление настройками блога',
         'delete_confirm' => 'Вы уверены, что хотите сделать это?',
         'chart_published' => 'Опубликовано',
         'chart_drafts' => 'Черновики',
         'chart_total' => 'Всего',
-        'new_category' => 'Новая категория'
+        'settings_description' => 'Управление настройками блога',
+        'show_all_posts_label' => 'Показывать все записи для внутренних (бэкенд) пользователей',
+        'show_all_posts_comment' => 'Показывать опубликованные и неопубликованные записи на фронтенде для внутренних (бэкенд) пользователей',
+        'tab_general' => 'Основные'
+    
```

Comment:
there's an extra newline here that's unnecessary.

Reply:
I did as in original file to match lines:
 https://github.com/rainlab/blog-plugin/blob/master/lang/en/lang.php

Delete empty line?

Reply:
Yes, it's incorrect :) No empty lines.

Reply:
Is done.